daget/__main__.py

```diff
@@ -22,22 +22,22 @@ def main():
   # get doi/url and resolve to landing page
   try:
     url = get_redirect_url(args.url)
+    res = [ele for ele in ["dataverse.harvard.edu", "dataverse.no", "snd.se/catalogue", "snd.gu.se", "su.figshare.com", "figshare.scilifelab.se", "zenodo.org"] if(ele in url)]
```

Comment:
checking the url on a hardcoded list would not cover all cases, all repositories where schema.org contains distrubution information will work and the list of figshare repositories is quite long: https://knowledge.figshare.com/type-of-client/institutions

A more generic solution would be to check if `get_file_list_from_repo()` returns any values.